writeup/summary.Rmd

```diff
@@ -51,6 +53,20 @@ daily <- load_local_data(date)
 # load results
 sgtf <- load_results(date)
 bias <- load_results(date, type = "bias")
+
+# get cases from correlated model
```

Comment:
wrapping as much of this as possible into helper functions will make eventual porting into the paper much easier